src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt

```diff
@@ -17,19 +15,39 @@ class AssessmentDto(
   val createdDate: LocalDateTime? = null,
 
   @Schema(description = "Completed Date", example = "2020-01-02T16:00:00")
-  val completedDate: LocalDateTime? = null
+  val completedDate: LocalDateTime? = null,
+
+  @Schema(description = "subject")
+  val subject: SubjectDto? = null
```

Comment:
just a single subject. for now we will use the first or null from the assessment until we figure it out what to do with the multiple subjects